benchmarks/run-benchmark.py

```diff
@@ -79,12 +87,20 @@
         help="Local path to delta repository which will be used for running the benchmark " +
              "instead of the version specified in the specification. Make sure that new delta" +
              " version is compatible with version in the spec.")
+    parser.add_argument(
+        "--cloud-provider",
+        choices=delta_log_store_classes.keys(),
+        help="Cloud where the benchmark will be executed.")
```

Comment:
since there is no default here, users must provide this right? have you made sure that the examples in the readme always provides this?

Reply:
Good catch. I'll update usage examples as well. Thanks!

Reply:
Updated.

Reply:
@tdas Could you re-review the PR, please? Thanks!